src/omen/cogs/wvw_commands.py

```diff
@@ -227,8 +308,13 @@ async def matchup(self, interaction: discord.Interaction):
                 )
                 embed_obj_value_string += f":{server}_circle: {camp_emoji}x{self.match_data_dict[server]['objectives']['camp']:=2} {tower_emoji}x{self.match_data_dict[server]['objectives']['tower']:=2} {keep_emoji}x{self.match_data_dict[server]['objectives']['keep']:=2} {castle_emoji}x{self.match_data_dict[server]['objectives']['castle']:=2}\n"
                 embed_skirmish_value_string += f":{server}_circle: Points: {self.match_data_dict[server]['skirmish']['skirmish_scores'][list(self.match_data_dict[server]['skirmish']['skirmish_scores'])[-1]]} (+{self.match_data_dict[server]['skirmish']['ppt']:=2} per tick)\n"
+                embed_predicted_victory_points_string += f":{server}_circle: Predicted Points: {self.match_data_dict[server]['predicted_victory_points']}\n"
+            #TODO Cleanly output when there's no prediction
+            embed_predicted_victory_points_string += f"Prediction Quality (lower is better): {self.match_data_dict['prediction_quality']}"
```

Comment:
![Prediction](https://user-images.githubusercontent.com/35041/216345258-cc2f45a6-4cf0-43cd-917f-7fb62a85caf0.png)
![No Prediction](https://user-images.githubusercontent.com/35041/216345266-32d37450-84cb-46b8-aa4b-262914f0eb52.png)

I also want to clean up what happens when there's no prediction. But it doesn't seem bad.